test/unit/warn-removed-experimental-config.test.ts

```diff
@@ -45,8 +45,9 @@ describe('warnOptionHasBeenMovedOutOfExperimental', () => {
     )
 
     expect(spy).toHaveBeenCalledWith(
-      expect.stringContaining('⚠'),
-      '`experimental.skipTrailingSlashRedirect` has been moved to `skipTrailingSlashRedirect`. Please update your next.config.js file accordingly.'
+      expect.stringContaining(
+        '`experimental.skipTrailingSlashRedirect` has been moved to `skipTrailingSlashRedirect`. Please update your next.config.js file accordingly.'
+      )
```

Comment:
Shouldn't we assert that `⚠` is still part of logs?

Reply:
The prefix is only part of the logger's styling feature, it's not the main focus of this test. It's also escaped here, you have to unescape to test the actual logging, which is not very related to the warning itself.

We could test the styles in logger's test, then when we update the styling the unrelated tests is not too much. I've experienced this before when we first time re-design the terminal output.

Reply:
I don't see how the styling is not part of this test. The styling conveys specific semantics and whether this is treated as an error or warning is relevant to the test.

Reply:
It's only intercepting the warning in the test, we know exact it's a warning. The only difference is the prefix

Reply:
Can you please restore the original assertion? I plan to work on this feature and don't want to have to manually verify things.

Reply:
added in [6af8b2a](https://github.com/vercel/next.js/pull/71939/commits/6af8b2a994f8fd5e0755ac54c04a65ca9a503394)